index.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+const priceData = {
+    Apples: '23.4',
+    BANANAS: '48',
+    oRAngGEs: '48.7584',
+  };
+  
+  let updatedPriceData = {};
+  for (let key in priceData){
+    let price = parseFloat(priceData[key]).toFixed(2);
+    updatedPriceData[key.toLowerCase()]= price;
```

Comment:
Все окей, але змінна price використовується лише тут, тому можна її не створювати взагалі, а зробити так:
```suggestion
    updatedPriceData[key.toLowerCase()] = parseFloat(priceData[key]).toFixed(2);
```